feedly/api_client/data.py

```diff
@@ -76,6 +76,24 @@ def tag_entries(self, entry_ids: List[str]):
         self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put',
                                     data={'entryIds': [entry_id for entry_id in entry_ids]})
 
+    def delete_tags(self, options: StreamOptions = None):
+        """
+        *** WARNING *** Non-reversible operation
+        Given a TagBas Streamable, remove tags corresponding to this tag stream, whichever user saved it, for
+        all articles downloaded with options StreamOptions.
+        :param options: specify high max_count to empty the board.
+        :return:
+        """
+        a_ids = [a["id"] for a in self.stream_contents(options)]
+        tag_id = self._get_id()
+        while len(a_ids) > 0:
+            batch_size = 50  # limitation due to the url length: articles are "de-tagged" by batch of 10.
```

Comment:
50 != 10 ?

Reply:
Ok I wrote 50 for batch_size in the comment as well.